### ISSUE TITLE
Experimentally integrate OOD detection to Antenna 

### ISSUE THREAD

**Issue by Yuyan-C** [RolnickLab/ami-ml#59]
We can add an OOD score to Antenna to show whether the species is in the training set or not.

Scores we can use include:
- maximum softmax probability (MSP)
- temperature scaling of MSP 
- RP-MSP 
- Energy score. 

The first three scores require the platform to save the softmax probability and the last score requires the platform to save logits. It is also useful to save the features. 

We can either use the global model or the regional model.

**Comment by mihow** [comment]
Woohoo!

**Comment by mihow** [comment]
Integrated and used for the Mount Totumas research trip in Panama. Deployed to a custom environment: https://ood.antenna.insectai.org/projects/11/summary

And available under the `ood` branch in the Antenna repo.

More experimentation will be done before integrating this into the main Antenna, or we will add it under a feature flag.
